providers/aws/aws_provider.py

```diff
@@ -290,7 +290,11 @@ def generate_regional_clients(service: str, audit_info: AWS_Audit_Info) -> dict:
     # Get json locally
     f = open_file(aws_services_json_file)
     data = parse_json_file(f)
-    json_regions = data["services"][service]["regions"][audit_info.audited_partition]
+    # Check if it is a subservice
+    if service == 'accessanalyzer':
+        json_regions = data["services"]['iam']["regions"][audit_info.audited_partition]
```

Comment:
Why it is needed to point the json regions to `iam` when the service is `accessanalyzer`?

Reply:
This is not a good practice, we are going to keep it by now but in case there are more cases like this we need to figure it out how to handle it